legal-api/src/legal_api/resources/business/business_filings.py

```diff
@@ -392,19 +392,32 @@ def _is_before_epoch_filing(filing_json: str, business: Business):
             filing_json['filing']['header']['date']).replace(tzinfo=datetime.timezone.utc)
         return filing_date < epoch_filing[0].filing_date
 
+    @staticmethod
+    def _is_historical_colin_filing(filing_json: str):
+        if filing_json['filing']['header']['source'] == 'COLIN' \
+                and filing_json['filing']['header']['date'] < '2019-03-08':
```

Comment:
Can this date be set somewhere?

Reply:
Created `constants.py` file and created a `BOB_DATE` constant which is now referenced in this method.